tmpfiles.d/eos-companion-app-helper.conf

```diff
@@ -0,0 +1,3 @@
+# Type Path    Mode UID  GID  Age Argument
+d /var/lib/companion-app-helper 0755 companion-app-helper companion-app-helper -
+Z /var/lib/companion-app-helper 0755 companion-app-helper companion-app-helper -
```

Comment:
How is this file installed? Does not seem to get installed from anywhere.

Reply:
Oops, I must have forgotten to add my changes to `Makefile.am`

Reply:
Okay, fixed 😅